Modules/_abc.c

```diff
@@ -710,6 +759,7 @@ _abc__abc_subclasscheck_impl(PyObject *module, PyObject *self,
         PyErr_SetString(PyExc_TypeError, "issubclass() arg 1 must be a class");
         return NULL;
     }
+    PyTypeObject *cls = (PyTypeObject *)self;
```

Comment:
I don't thuink you need this elsewhere, so just use `_PyType_GetDict((PyTypeObject *)self)`